---------------------
PatchSet 325 
Date: 2010/09/28 09:25:02
Author: millert
Branch: HEAD
Tag: (none) 
Log:
pregenerate unicode files; mktables uses too much memory for some platforms

Need to run something like:
$(MINIPERL) $(Icwd) lib/unicore/mktables -C lib/unicore -P pod -maketest -makelist -p

Index: src/gnu/usr.bin/perl/Makefile.SH
diff -u src/gnu/usr.bin/perl/Makefile.SH:1.20 src/gnu/usr.bin/perl/Makefile.SH:1.21
--- Makefile.SH:1.20	Fri Sep 24 15:06:39 2010
+++ Makefile.SH	Tue Sep 28 16:25:02 2010
@@ -995,10 +995,8 @@
 unidatafiles $(unidatafiles) pod/perluniprops.pod: uni.data
 
 uni.data: $(MINIPERL_EXE) $(CONFIGPM) lib/unicore/mktables $(nonxs_ext)
-	$(MINIPERL) $(Icwd) lib/unicore/mktables -C lib/unicore -P pod -maketest -makelist -p
-#	Commented out so always runs, mktables looks at far more files than we
-#	can in this makefile to decide if needs to run or not
-#	touch uni.data
+#	$(MINIPERL) $(Icwd) lib/unicore/mktables -C lib/unicore -P pod -maketest -makelist -p
+	touch uni.data
 
 # $(PERL_EXE) and ext because buildtoc uses Text::Wrap uses re
 # But also this ensures that all extensions are built before we try to scan
